### ISSUE TITLE
Regarding "Just My Code" support

### ISSUE THREAD

**Issue by TryerGit** [microsoft/vscode-cpptools#6668]
Hello,

I was a bit surprised to see the following thread declared closed:

[Issue #6232](https://github.com/microsoft/vscode-cpptools/issues/6232)

According to this [Visual Studio IDE document](https://docs.microsoft.com/en-us/visualstudio/debugger/just-my-code?view=vs-2019), enabling "Just My Code" is achieved via a compiler option when running `CL.exe`. In any case, `CL.exe` is part of the compiler toolchain that is outside of VSCode anyway. 

So, if VSCode is setup with a build task that calls `CL.exe` and `MSBuild.exe` where `CL.exe` is provided with a `.vcxproj` project file that has the compiler option `/JMC` set, then why is it stated on issue 6232 linked above that Just My Code is not supported by the Cpptools extension?

In other words, what would it mean for Cpptools extension to support "Just My Code" ?

It appears that skipping uninteresting code while debugging is provided for Node.js debugging ( see [link](https://code.visualstudio.com/docs/nodejs/nodejs-debugging) )

while currently unavailable for C++ debugging, although, as pointed to above, what that means is not clear.

**Comment by sean-mcmanus** [comment]
It was closed as a duplicate of https://github.com/microsoft/vscode-cpptools/issues/5763 .

I would guess there could be additional debugger work required.

@WardenGnaw might know more.

**Comment by WardenGnaw** [comment]
When compiling with `/JMC` it inserts helper function calls to `__CheckForDebuggerJustMyCode`. The debugger needs to support  calls that hit that method and step out until it reaches a user code frame.

Please vote on https://github.com/microsoft/vscode-cpptools/issues/5763 by adding a 👍 to help us prioritize our features.
